### ISSUE TITLE
Question about STEP 4

### ISSUE THREAD

**Issue by jlong2git** [smrizzo2/mapit#1]
Did you manual create 200+ images for each entry in sierra_maps_db.php or did you automate the placement of the marker ?

**Comment by smrizzo2** [comment]
Yes, the images were created manually, but you could automate it.
